src/plugins/charts/public/services/colors/color_palette.ts

```diff
@@ -65,8 +65,8 @@ export function createColorPalette(num: number): string[] {
     throw new TypeError('ColorPaletteUtilService expects a number');
   }
 
-  const colors = seedColors;
```

Comment:
This whole function (`createColorPalette`) may be able to be removed in favor of directly calling `euiPaletteColorBlind()`. The difference is that this function currently takes the desired number of colors as a param, while `euiPaletteColorBlind` takes a number of rotations instead.

So we can either:

1. Keep `createColorPalette` but have it return the first `num` values of `euiPaletteColorBlind()` with a sufficient number of rotations or
2. Just refactor callers of `createColorPalette` call `euiPaletteColorBlind()` with rotations instead of a total number of colors

I have a hunch it will be fairly quick to do 2, but am also fine with doing 1 in this PR, and 2 as a subsequent follow-up.

Don't take my word for it, but I _think_ it's only called from https://github.com/opensearch-project/OpenSearch-Dashboards/blob/84ec854c94cc11a84272f87f8081fc5097720af3/src/plugins/charts/public/services/colors/mapped_colors.ts#L103

Reply:
On second thought, let's just do the easy way for now, and do the larger refactoring as part of https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4320

Reply:
Agree! `createColorPalette` method will be removed/refactored as part of #4320.